make/hotspot/lib/CompileJvm.gmk

```diff
@@ -144,6 +144,8 @@ $(call FillFindCache, $(JVM_SRC_DIRS))
 ################################################################################
 # Now set up the actual compilation of the main hotspot native library
 
+# TODO: Should merge once JDK-8294314 is in.
```

Comment:
Left-over comment should be removed.

Reply:
I shall, once JDK-8294314 integrates. I am keeping those build changes to force a merge conflict.